lib/rubocop/cop/style/parentheses_around_condition.rb

```diff
@@ -71,6 +70,16 @@ def message(node)
           article = kw == 'while' ? 'a' : 'an'
           "Don't use parentheses around the condition of #{article} `#{kw}`."
         end
+
+        def parens_allowed?(cond)
```

Comment:
I wonder where `cond`, `node` or `cond_node` is the best param name here, but that's not a big deal.